docs/Reference/CLI/CLI-Syntax.md

```diff
@@ -4,40 +4,50 @@ description: Teku command line interface reference
 
 # Teku command line
 
-This reference describes the syntax of the Teku Command Line Interface (CLI) options and subcommands.
+This reference describes the syntax of the Teku command line interface (CLI) options.
 
 !!! important
```

Comment:
This was added when Teku was still in the early stages of development. Perhaps it's time to remove it?